custom/bin/ARE/web/webapps/asterics-microcontroller-arduino/lib/asterics-microcontroller-arduino.js

```diff
@@ -12,7 +12,7 @@ function defaultSuccessCallback(data, status) {
 }
 function defaultErrorCallback(status, e) { 
     // alert("Error message: "+ e);
-    alert("Connection with ARE failed.\n Check if your ARE is running on your machine and retry.");
+    // alert("Connection with ARE failed.\n Check if your ARE is running on your machine and retry.");
```

Comment:
Maybe add console.log instead of alert function. Just to now, if we got into the callback or not.